app/bundles/ReportBundle/Scheduler/Model/SchedulerPlanner.php

```diff
@@ -63,12 +63,15 @@ private function planScheduler(Report $report)
 
     private function removeSchedulerOfReport(Report $report)
     {
-        $scheduler = $this->schedulerRepository->getSchedulerByReport($report);
-        if (!$scheduler) {
+        $schedulers = $this->schedulerRepository->findBy(['report' => $report]);
```

Comment:
Can you marek the `getSchedulerByReport` method as deprecated as it's not used from anywhere else?

Reply:
Done